lib/connection.js

```diff
@@ -24,12 +24,53 @@ var EndpointState = require('./endpoint.js');
 var Session = require('./session.js');
 var Transport = require('./transport.js');
 
+var fs = require('fs');
+var os = require('os');
+var path = require('path');
```

Comment:
What is the impact on browserification by using these node.js modules?

Reply:
This behaviour is not intended for use in the browser. It is only triggered by a connect() call with no argument passed in. That does not work in the browser anyway, as the browser needs to hook in the websocket. 

The browserified code will however continue to work fine in the browser though this feature cannot be used there.